kafka-broker/kafka-broker-install.sh

```diff
@@ -2,13 +2,20 @@
 
 ## Script to install a kafka & zookeeper broker
 
-# Install Java
 sudo yum update -y
 sudo yum install -y java-1.8.0-openjdk-devel
 
 # Install Amazon SSM agent
 sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
 
+# Install python-pip
+# pip is not available in CentOS 7 core repositories there is a requirement to enable EPEL repositories prior
```

Comment:
Can you add that gcc and python-devel are required to enable the `twofish` indirect dependency of `acm-pca-cert-generator` to be built and installed please?